File: Facebook.NET/Contracts/GraphAPI.Job.cs
using System;
using System.Runtime.Serialization;

namespace Facebook
{
    public static partial class GraphApi
    {
        [DataContract]
        public sealed class Job
        {
            [DataMember(Name = "employer")]
            public GraphApi.Domain Employer { get; set; }

            [DataMember(Name = "position")]
            public GraphApi.Domain Position { get; set; }

            [DataMember(Name = "description")]
            public string Description { get; set; }

            [DataMember(Name = "start_date")]
            public string StartDate { get; set; }

            [DataMember(Name = "end_date")]
            public string EndDate { get; set; }
        }
    }
}

File: Facebook.NET - VB/Contracts/GraphAPI.Job.vb
Imports System
Imports System.Runtime.Serialization

Namespace Facebook
    Public NotInheritable Partial Class GraphApi
        <DataContract()>
        Public NotInheritable Class Job
            <DataMember(Name:="employer")>
            Public Property Employer() As GraphApi.Domain
            <DataMember(Name:="position")>
            Public Property Position() As GraphApi.Domain
            <DataMember(Name:="description")>
            Public Property Description() As String
            <DataMember(Name:="start_date")>
            Public Property StartDate() As String
            <DataMember(Name:="end_date")>
            Public Property EndDate() As String
        End Class
    End Class
End Namespace